sdk/mixedreality/mixed-reality-authentication/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 1.0.0-beta.2 (2022-01-12)
+
+- Package does not longer depend on `@azure/core-http`. Now, it depends on `@azure/core-client` and `@azure/core-rest-pipeline` instead.
```

Comment:
Can you rephrase this sentence? Something similar to "Moved the package dependency to core V2 (`azure/core-client` and `@azure/core-rest-pipeline` packages`) from `@azure/core-http`)

Reply:
Yes, Change committed